### DISCUSSION TITLE
Sugerencia

### DISCUSSION THREAD

**Discussion by C4rp3nt3rArg** [IvanMass99/Calculadora-Gamer#5] [Ideas]
Hola, podrías agregar la opción de calcular en EA Games?... es que juego a los sims y para ir calculando cuanto me cuestan las expansiones y he visto que EA tiene otros juegos, quizás le es útil a alguien más. 

Por cierto, muchísimas gracias por tu calculadora

**Comment by IvanMass99** [comment]
¡Hola! Este sábado saldrá una nueva actualización corrigiendo algunos detalles y agregando la tienda de EA. Saludos!
